src/drive/storage/index.ts

```diff
@@ -106,16 +106,22 @@ export class Storage {
    * @param {number} folderId - ID of the folder.
    * @param {number} [offset=0] - The position of the first file to return.
    * @param {number} [limit=50] - The number of files to be returned.
+   * @param {string} [sort=plainName] - The reference column to sort it.
+   * @param {string} [order=ASC] - The order to be followed.
    * @returns {[Promise<FetchPaginatedFolderContentResponse>, RequestCanceler]} An array containing a promise to get the API response and a function to cancel the request.
    */
   public getFolderFiles(
     folderId: number,
     offset = 0,
     limit = 50,
+    sort = '',
+    order = '',
   ): [Promise<FetchPaginatedFolderContentResponse>, RequestCanceler] {
     const offsetQuery = `/?offset=${offset}`;
     const limitQuery = `&limit=${limit}`;
-    const query = `${offsetQuery}${limitQuery}`;
+    const sortQuery = `&sort=${sort}`;
```

Comment:
If the sort/order are not going to have any value, do not add it to the request, please

Reply:
done